operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,114 @@
+# Audit Logs
+
+## Overview
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+
+By default, host and Kubernetes API audit logging are not enabled.
+It is not required for both to be enabled or disabled at the same time.
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following options based on the installation status of the system:
+
+   For each of the following options, only enable the desired level of audit logging.
+   It is not required to enable both.
+
+   * During CSM install, from the PIT node
+
+     To update the audit log settings during the installation, use one of the following options:
+
+     * **Option 1**
+
+       During the installation, audit logging is enabled or disabled by modifying the CSI settings.
```

Comment:
This option needs to clarify that this needs to happen at the same time that "csi config init" is normally run during the install. I think that is obvious to someone who knows how the installs work pretty well, but I don't think everyone would realize that they couldn't do this step later on in the install.

And really, we should probably add a link from that part of the install procedure over to here, otherwise nobody is going to find it in that situation.
https://github.com/Cray-HPE/docs-csm/blob/release/1.0/install/prepare_configuration_payload.md

And these CSI options should be added to this table:
https://github.com/Cray-HPE/docs-csm/blob/release/1.0/install/prepare_configuration_payload.md#command-line-configuration-payload

Reply:
These two amendments are specific to 1.0 and 1.2.
For 1.3 and 1.4 that doesn't exist, but this should be mentioned during https://github.com/Cray-HPE/docs-csm/blob/main/install/pre-installation.md#33-customize-system_configyaml for example it could be added to one of the existing notes "Enable/disable audit logging, refer to here for background info"

Reply:
I'm confused. Those CSI arguments exist in 1.0 and 1.2 but not later? Or those CSI arguments exist in 1.3 and 1.4, but not earlier?

Right now it looks like all versions of this PR mention them, unless I am missing something.

Reply:
I believe he meant that the prepare_configuration_payload.md file only exists in 1.0 and 1.2, and that the references in the install doc will need to be in a different location for 1.3 and 1.4.

Reply:
Correct, it only exists in 1.0 and 1.2. In 1.3 and 1.4 the page was refactored and consolidated into this step: https://github.com/Cray-HPE/docs-csm/blob/release/1.3/install/pre-installation.md#3-create-system-configuration